HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.1.9
+++++++
+* azdev extension publish: add `storage-account-ley`
```

Comment:
typo for ”ley“？
Actually it should be “add --storage-acocunt-key and remove --storage-subscription".

Reply:
Fixed